cumulus/parachains/runtimes/glutton/glutton-kusama/src/lib.rs

```diff
@@ -204,12 +232,18 @@ construct_runtime! {
 			Pallet, Call, Config<T>, Storage, Inherent, Event<T>, ValidateUnsigned,
 		} = 1,
 		ParachainInfo: parachain_info::{Pallet, Storage, Config<T>} = 2,
+		// TODO: probably not needed now, but I'm not sure
+		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 3,
```

Comment:
Should be needed for Aura, yes
```suggestion
		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 3,
```

Reply:
Yes I figured it out later but forgot to remove this comment.